Modules/Core/SpatialObjects/include/itkMeshSpatialObject.h

```diff
@@ -96,7 +96,10 @@ class ITK_TEMPLATE_EXPORT MeshSpatialObject : public SpatialObject<TMesh::PointD
 
 #if !defined(ITK_LEGACY_REMOVE)
   /** \deprecated Return the type of pixel used */
-  itkLegacyMacro(const char * GetPixelTypeName()) { return m_PixelType.c_str(); }
+  itkLegacyMacro(const char * GetPixelTypeName())
```

Comment:
Unrelated. Preferably group with the other into a separate commit.

Reply:
- Resolved (again) by pull request #4857